PORTAL/jobs/_pyperformance.py

```diff
@@ -1568,6 +1567,7 @@ def split_benchmarks(self) -> Mapping[SuiteType, "PyperfResults"]:
             results = self._copy()
             results._data = data
             results._by_suite = {suite: data['benchmarks'][0]}
```

Comment:
For the sake of `copy_to()`:

```suggestion
            results._by_suite = {suite: data['benchmarks'][0]}
            results._modified = True
```

And, of course, we'd need to set the default to `False`.